crates/runtime/src/continuation.rs

```diff
@@ -315,7 +320,12 @@ pub fn cont_new(
             .map_err(|error| TrapReason::user_without_backtrace(error.into()))?;
         let args_ptr = payload.data;
         let fiber = Fiber::new(stack, move |_first_val: (), _suspend: &Yield| unsafe {
-            (func_ref.array_call)(callee_ctx, caller_ctx, args_ptr.cast::<ValRaw>(), capacity)
+            (func_ref.array_call)(
+                callee_ctx,
+                caller_ctx,
+                args_ptr.cast::<ValRaw>(),
```

Comment:
We should document here that it is crucial that the lifetime of `args_ptr` outlives the activation frame. This should always be true in our current setup as we only deallocate after a continuation has returned successfully and all of its data has been copied.